auditlog/audit_log_record.go

```diff
@@ -0,0 +1,100 @@
+package auditlog
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"reflect"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/fabric8-services/fabric8-common/convert"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+// Record an audit log record to track usage of the service
+type Record struct {
```

Comment:
I would call this type just `AuditLog`, and rename the file to `audit_log.go`.  Also, I would try to follow the same structures we have established in the auth service, i.e. repository objects should be placed within a `repository` directory.

Reply:
ok, let me do that

Reply:
on second thought, I'm not sure we should introduce a `repository` sub-pkg yet, since there's no `service` layer at the moment (the controller handles the transaction at this point)

Reply:
done (partially - no `repository` subpkg yet) in b78e640